src/views/Home/components/LostAndFoundCard/index.tsx

```diff
@@ -0,0 +1,221 @@
+          .sort((a, b) => new Date(b.dateCreated).getTime() - new Date(a.dateCreated).getTime())
+          .map((report) => ({
+            ...report,
+            colors: report.colors || [], // Ensure colors is an array
+          }));
+
+        setActiveReports(active);
+      } catch (error) {
+        console.error('Error fetching missing items:', error);
+      } finally {
+        setLoading(false);
+      }
+    };
+    fetchMissingItems();
+  });
+
+  const renderReportContent = (report: MissingItemReport) => (
+    <>
+      <Grid item xs={width >= 900 ? 4 : 12}>
+        <Typography align="center">{formatDate(report.dateLost)}</Typography>
+      </Grid>
+      <Grid item xs={width >= 900 ? 4 : 12}>
+        <Typography
+          align="center"
+          style={{ cursor: report.locationLost.length > 15 ? 'pointer' : 'default' }}
+        >
+          {report.locationLost.length <= 15
+            ? report.locationLost
+            : `${report.locationLost.slice(0, 15)}...`}
+        </Typography>
+      </Grid>
+      <Grid item xs={width >= 900 ? 4 : 12}>
+        <Typography align="center">{report.category}</Typography>
+      </Grid>
+    </>
+  );
+
+  return (
+    <Card>
+      <CardHeader
+        title={
+          <Grid container direction="row">
+            <Grid item xs={7}>
+              Lost and Found
+            </Grid>
+            <Grid item xs={5}>
+              <Button variant="contained" color="secondary" component={Link} to="/LostAndFound">
+                Lost and Found Page
+              </Button>
+            </Grid>
+          </Grid>
+        }
+        className="gc360_header"
+      />
+      <CardContent>
+        <Grid container direction="row" justifyContent="space-between">
+          <Grid item xs={6}>
+            <Button
+              variant="contained"
+              color="secondary"
+              component={Link}
+              to="/LostAndFound/missingitemform"
+            >
+              Report Lost
+            </Button>
+          </Grid>
+          <Grid item xs={6}>
+            <Button variant="contained" color="secondary" component={Link} to="/LostAndFound">
+              Report Found
+            </Button>
+          </Grid>
+        </Grid>
+        {activeReports?.length < 1
+          ? noReports
+          : width >= 900 && (
+              <Grid container spacing={2} className={styles.headerRow}>
+                <Grid item xs={4}>
+                  <Typography align="center">Date Lost</Typography>
+                </Grid>
+                <Grid item xs={4}>
+                  <Typography align="center">Location Lost</Typography>
+                </Grid>
+                <Grid item xs={4}>
+                  <Typography align="center">Description</Typography>
+                </Grid>
+              </Grid>
+            )}
+        {activeReports?.length > 3 ? (
+          <>
+            {activeReports.slice(0, 3).map((report) => (
```

Comment:
I don't actually think we have to do the slicing and checking here.  You can instead do the slice in the useEffect where you first load the data, and just set a flag to whether you had to cut down the length, to use when determining whether to display the "more items not shown".